CONTRIBUTING

```diff
@@ -0,0 +1,34 @@
+QLever is licensed under the Apache License, Version 2.0. The copyright is
+jointly held by the authors of the code (each for their own contributions).
+This is true regardless of whether this is acknowledged in the individual files
+of the source code or not.
+
+Nevertheless, for the sake of clarity and acknowledgment, we write the
+following at the top of each file of the codebase (from least to most recent):
+
+// Copyright {year of year span} {name and email of copyright owner}
+// Copyright {year of year span} {name and email of copyright owner}
+// Copyright {year of year span} {name and email of copyright owner}
```

Comment:
This is confusing. None of the files look like this currently, they use the variation mentioned waaay below.